eclipse-collections-code-generator/src/main/resources/api/list/primitiveList.stg

```diff
@@ -125,6 +131,20 @@ public interface <name>List extends Reversible<name>Iterable
     {
         throw new UnsupportedOperationException("Default method to prevent breaking backwards compatibility");
     }
+<if(primitive.specializedStream)>
+
+    Spliterator.Of<name> primitiveSpliterator();
```

Comment:
If we called this spliterator instead of primitiveSpliterator, would that cause a naming clash?

Is there some other reason to include the word primitive in the name? It feels redundant to me inside a primitive interface.

Reply:
i thought that in the future you will want to make a spliterator with boxed values, to be consistent with the object streams.

Reply:
In the same way that primitive streams have a method boxed(), I could imagine we'd add boxed() to primitive iterables that would return RichIterable. I doubt we'd want individual methods to have primitive/boxed variants. Thoughts?